ls1intum/Artemis
# Follow-up refactoring - adapt structure for multi organizations

### Checklist

### Motivation and Context


Follow-up changes from https://github.com/ls1intum/Artemis/pull/2748

### Description

Some refactoring missing from https://github.com/ls1intum/Artemis/pull/2748


### Steps for Testing


1. Log in into Artemis as Admin
2. Go to Organization Management
3. Update an organization
4. No errors should appear

### Test Coverage




### Screenshots
